ontology/yaml/resources/units/units.yaml

```diff
@@ -204,6 +204,9 @@ pressure:
 - pounds_force_per_square_inch
 - bars
 - millibars
+pulserate:
+- beats_per_minute: STANDARD
```

Comment:
whats the difference between a beat per minute and a pulse per minute

Reply:
Done the changes as context here is different for particular waves

Reply:
Hi Claire ,
There were certain mistakes in previous commits .
On basis of recent inputs from you and the manufacturer would request you to review it.